index.js

```diff
@@ -283,6 +289,12 @@ function choosePluralForm(text, locale, count) {
   return text;
 }
 
+// Determines if an object passes as a smart_count parameter for
+// pluralization is valid
+function isValidSmartObject(count) {
+  return (typeof count.count === 'number' && !!count.value);
```

Comment:
Right, the parenthesis are useless in this case. I kind of considered some type safety for count.value but I see your point. I would suggest replacing this line with 

`return typeof count.count === 'number' && count.value.toString()`

if this is okay with you, and a better explanation in the comment perhaps.
Do you think it is ok if I write some negative unit tests as well to cover those cases?